src/scripts/findOffers.ts

```diff
@@ -1,9 +1,179 @@
-const findOffers = () => {
-    console.log('Scrapping...')
+import puppeteer, { Browser, Page, ElementHandle } from 'puppeteer';
+import { Cluster } from 'puppeteer-cluster';
+import ScraperOptions from '../interfaces/ScraperOptions';
+import JobOffer from '../interfaces/JobOffer';
+import writeToJson from '../helpers/writeToJson';
 
-    setTimeout(() => {
-        console.log('10 offers found.')
-    }, 3000)
+const pptOptions = {
+  headless: false,
+  args: [`--window-size=1920,1080`, '--no-sandbox'],
+};
+
+const selector = 'div.css-ic7v2w';
+
+async function delay(miliseconds: number): Promise<void> {
+  return new Promise(function (resolve) {
+    setTimeout(resolve, miliseconds);
+  });
+}
+
+async function navigateTo(url: string, page: Page): Promise<void> {
```

Comment:
**nitpick:** Preferably such functions like `delay` or `navigateTo` should not be a part `findOffers.ts` because these are helper functions for your scrapper that can be re-used independently